easycv/hooks/tensorboard.py

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) Alibaba, Inc. and its affiliates.
+import numpy as np
+import torch
+from mmcv.runner.dist_utils import master_only
+from mmcv.runner.hooks import HOOKS
+from mmcv.runner.hooks import TensorboardLoggerHook as _TensorboardLoggerHook
+
+
+@HOOKS.register_module()
+class TensorboardLoggerHookV2(_TensorboardLoggerHook):
+
+    def visualization_log(self, runner):
+        visual_results = runner.visualization_buffer.output
+        images = visual_results.get('images', [])
```

Comment:
the data format in visualization_buffer can be simplified,  we need a group of images, display name for each image, and the group name for the group

* in tensorboard, group_name/image_name as the key for summary
* in wandb,  group_name can be used as key of data dict